Dockerfile

```diff
@@ -1,57 +0,0 @@
-FROM ubuntu:latest@sha256:3f119dc0737f57f704ebecac8a6d8477b0f6ca1ca0332c7ee1395ed2c6a82be7
-
-ENV NODE_VERSION 8.11.2
-
-LABEL maintainer="Vincent Voyer <vincent@zeroload.net>"
-
-ENV LC_ALL=C
-ENV DEBIAN_FRONTEND=noninteractive
-ENV DEBCONF_NONINTERACTIVE_SEEN=true
-
-EXPOSE 4444
-
-RUN apt-get -qqy update
-RUN apt-get -qqy install \
-    apt-transport-https \
-    ca-certificates \
-    curl \
-    software-properties-common \
-    sudo
-RUN curl -sS -o - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -
-RUN echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list
-RUN echo "deb http://archive.ubuntu.com/ubuntu xenial main universe\n" > /etc/apt/sources.list \
-  && echo "deb http://archive.ubuntu.com/ubuntu xenial-updates main universe\n" >> /etc/apt/sources.list \
-  && echo "deb http://security.ubuntu.com/ubuntu xenial-security main universe\n" >> /etc/apt/sources.list
-RUN apt-get -qqy update
-
-
-RUN groupadd --gid 1000 node \
-  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
-RUN echo 'node ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
-
-RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
-RUN apt-get -qqy --no-install-recommends install \
-  nodejs \
-  firefox \
-  google-chrome-stable \
-  openjdk-8-jre-headless \
-  xvfb \
-  xfonts-100dpi \
-  xfonts-75dpi \
-  xfonts-scalable \
-  xfonts-cyrillic
-
-RUN export DISPLAY=:99.0
-RUN Xvfb :99 -shmem -screen 0 1366x768x16 &
-
-WORKDIR /home/node
-# For development
-# ADD . ./selenium-standalone-local
-# RUN chown node:node -R .
-USER node
-RUN npm init -y
-# RUN npm install -i ./selenium-standalone-local
-RUN npm install -i selenium-standalone
-
-
-CMD DEBUG=selenium-standalone:* ./node_modules/.bin/selenium-standalone install && DEBUG=selenium-standalone:* ./node_modules/.bin/selenium-standalone start
```

Comment:
Can you specify on the task what was wrong with this approach or what errors it issued so I can replicate?

Reply:
I do not consider the previous approach wrong. I just prefer global installation as it eases understanding of the Dockerfile process. Also it is closer to what a normal user would do locally on its machine.

Reply:
@catalan-adobe the only problem I had with the current configuration is that it issued a warning about chrome driver, but the service starts and the tests work https://github.com/serbanghita/webdriverio-docker

Reply:
What warning?

Reply:
@catalan-adobe I mean previous configuration: `error in fs.access /home/node/node_modules/selenium-standalone/.selenium/chromedriver/2.40-x64-chromedriver { Error: EACCES: permission denied, access '/home/node/node_modules/selenium-standalone/.selenium/chromedriver/2.40-x64-chromedriver'` - but as far as I investigated this was a race condition in JS which can be fixed with refactori

Reply:
@catalan-adobe what was not working on the previous image? I ran it and it's starting Selenium on 4444

Reply:
Correct. But then you cannot start browsers.

Reply:
@catalan-adobe thanks for updating the description about the errors. I'll recheck